sport/app/rugby/conf/RugbyLifecycle.scala

```diff
@@ -22,12 +22,12 @@ class RugbyLifecycle(
   override def start(): Unit = {
     jobs.deschedule("FixturesAndResults")
     jobs.schedule("FixturesAndResults", "5 0/30 * * * ?") {
-      rugbyStatsJob.fetchFixturesAndResults()
+      rugbyStatsJob.fetchFixturesAndResults
```

Comment:
I don't think they are related to play `2.6 but `()` is used in `scala` on methods that have a side effect and therefore should not be removed here I think, see https://docs.scala-lang.org/style/naming-conventions.html#parentheses

Reply:
Fair point. 
However it means method that only takes implicit parameters need to be declare this way:
```
def myMethod()(implicit something: Something)
```
which is quite ugly